styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesPodEventTranslator.java

```diff
@@ -68,6 +68,8 @@ public TerminationLogMessage(
   }
 
   private static Optional<Integer> getExitCode(Pod pod, ContainerStatus status, Stats stats) {
+    final String workflowInstance = pod.getMetadata().getAnnotations()
+        .getOrDefault(KubernetesDockerRunner.STYX_WORKFLOW_INSTANCE_ANNOTATION, "N/A");
```

Comment:
`pod` is used for other purpose in this method however. do we really want to add another param as signature?